docs/cloud/pouta/launch-vm-from-web-gui.md

```diff
@@ -157,6 +157,20 @@ Click the plus to create a new IP, select the IP, select your machine under **In
 **Figure** Floating IP association dialog
 
 !!! Tip
-    Please note: Allocated and assigned floating IPs are billed at the rate of 0,2 BU/hr. You can additionally read our [blog post](http://pouta.blog.csc.fi) for management of floating IPs in a cPouta project.
+    Please note: Allocated and assigned floating IPs are billed at the rate of 0,2 BU/hr. You can additionally read our [blog post](http://cloud.blog.csc.fi/2017/12/floating-ip-management.html) for management of floating IPs in a cPouta project.
+
+## Adding a security groups to the machine in cPouta
```

Comment:
I'm going to approve this already but can you still fix

```
-a security groups
+security groups
```

Reply:
Done